win32ss/gdi/ntgdi/region.c

```diff
@@ -2149,9 +2149,9 @@ REGION_bXformRgn(
             /* Loop all rects in the region */
             for (i = 0; i < prgn->rdh.nCount - 1; i++)
             {
-                for (j = i; i < prgn->rdh.nCount; i++)
+                for (j = i; j < prgn->rdh.nCount; j++)
                 {
-                    NT_ASSERT(prgn->Buffer[i].top < prgn->Buffer[i].bottom);
+                    NT_ASSERT(prgn->Buffer[i].top <= prgn->Buffer[i].bottom);
```

Comment:
Change looks good, but also: this assert does not depend on `j` so it can be moved outside the inner loop.

Reply:
Should one of these `i` be a `j`?
If not, then this check is even redundant/superfluous, after `RECTL_vMakeWellOrdered()` call.

Note that this change is expected to be temporary, as CORE-12204 should remove empty rectangles...